mopidy/mpd/translator.py

```diff
@@ -97,6 +97,12 @@ def track_to_mpd_format(track, position=None, stream_title=None):
     if track.musicbrainz_id is not None:
         result.append(('MUSICBRAINZ_TRACKID', track.musicbrainz_id))
 
+    if track.album and track.album.uri:
+        result.append(('X-AlbumUri', track.album.uri))
+    if track.album and track.album.images:
+        images = ';'.join(i for i in track.album.images if i is not '')
```

Comment:
This is technically correct, but https://docs.mopidy.com/en/latest/api/core/?highlight=get_images#mopidy.core.LibraryController.get_images is replacing it. So if you want to continue getting image data your probably want to use that instead.

I think we can merge this as is, but I would do a followup PR or bug with adding and images argument to this function which takes the results from a `get_images` call.